types/dragino/cpl01/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "CPL01",
+  "version": "1.0.0",
+  "manufacturer": "Dragino",
+  "url": "https://www.dragino.com/products/lorawan-nb-iot-door-sensor-water-leak/item/205-cpl01.html",
+  "description": "The Dragino CPL01 is a LoRaWAN Contact Sensor. It detects open/close status and uplinks to IoT server via LoRaWAN network. user can see the dry contact status, open time, open counts in the IoT Server.",
```

Comment:
typo: It is capable to measure the dry contact status, open time and open counts. ?